Formula/gradle-profiler.rb

```diff
@@ -0,0 +1,26 @@
+class GradleProfiler < Formula
+  desc "Profiling and benchmarking tool for Gradle builds"
+  homepage "https://github.com/gradle/gradle-profiler/"
+  url "https://repo.gradle.org/gradle/ext-releases-local/org/gradle/profiler/gradle-profiler/0.12.0/gradle-profiler-0.12.0.zip"
+  sha256 "0c8f6a8e9f62860e5ea675534057306d0e4a9b9607353338bab978b4fa0c8a17"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install %w[bin lib]
+    (bin/"gradle-profiler").write_env_script libexec/"bin/gradle-profiler",
+      :JAVA_HOME => "${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
+  end
+
+  test do
+    (testpath/"build.gradle").write <<~EOS
```

Comment:
I think you should add an (empty) `settings.gradle` as well, so Gradle doesn't try to search upwards for one.

Reply:
Can do. For what it's worth this is copied from the `gradle.rb` formula. I will fix it separately.

Reply:
Awesome, thank you!

Reply:
Fixed here and for Gradle in https://github.com/Homebrew/homebrew-core/pull/58482